localization/captureFallback-new.xlf

```diff
@@ -2443,18 +2443,6 @@
             <target>Sim</target>
          </segment>
       </unit>
-      <unit id="transcriber.aheadTip">
```

Comment:
I am not sure how this file is used? was it just a temporary file?

Reply:
I have no idea. There are so many localization files now, and directories and I'm not sure what needs to be updated.  When I delete the az, hi, nl, tl directories, they come back.  It makes it harder to find which localization file I want.

Reply:
I removed this file